tests/lac/schur_complement_05.with_cxx11=on.with_trilinos=true.with_mpi=true.mpirun=1.output

```diff
@@ -6,10 +6,10 @@ DEAL:ManualSchur:cg::Convergence step 11 value 0
 DEAL:SchurComplementOp:Pre::Starting value 12.7149
 DEAL:SchurComplementOp:Pre::Convergence step 43 value 1.05660e-05
 DEAL:SchurComplementOp:Main::Starting value 7.49608e-06
-DEAL:SchurComplementOp:Main::Convergence step 12 value 4.00858e-10
+DEAL:SchurComplementOp:Main::Convergence step 22 value 2.93110e-10
```

Comment:
This looks dodgy to me. I'm still investigating why the number of solver iterations has increased.